rules.json

```
@@ -2,16 +2,167 @@
+"• Directors/Administration always have the final say in any decision made.",
+
+            "Policy C: Respect Staff and Their Decisions",
+"• Staff Members always have the final say, no point in arguing a punishment. Instead, accept the consequences and if you still feel like it was treated poorly or you didn't do anything, fill out a Staff Report and appeal your punishment.",
+"• Do NOT question decisions made by the Staff Team of TXDPSRP.",
+"• If you believe there is a problem with the Staff Team of TXDPSRP, bring it up the CoC (Chain of Command), until it finally reaches the Head Administration.",
+"• If you believe there is a problem with a specific staff member, fill out the Staff Report form located on our forums, and we will take care of the issue, if necessary.",
+"• Always be honest with the Staff Team of TXDPSRP.",
+
+
+
+
+
+
+
+            
+
+    },
+    {
+        "title": "Forum Rules",
+        "isChild": false,
+        "rules": [
+         "• Respect and treat everyone EQUALLY. This means to treat and respect both members and the staff team how you would like to be treated.",
+         "• Maturity must be maintained at ALL TIMES.",
+         "• Do not impersonate ANY other members on the Forums. This also applies to any known YouTubers/Celebrities. This falls under “Intentionally Deceiving,” and you can be punished for it.",
+         " • Do not reply to ANY staff related questions. Wait until a staff member responds to that question.",
+         "• If you believe a player has broken the rules, file a player report form located here, and our Reports Team will take care of the issue, if necessary.",
+         "• If you believe a forum member has broken the rules, click the report button on the comment, and a staff member will take care of it, if necessary.",
+         "• If you discover a bug, do NOT abuse/exploit it as you will be severely punished. Instead, please file a bug report here so we can fix it as soon as possible.",
+         "• Do NOT have any inappropriate or NSFW profile pictures.",
+         "• If you believe there is a problem with a specific staff member, fill out the Staff Report form located here, and we will take care of the issue, if necessary.",
+         "• Do NOT make any false reports, such as player reports, staff reports or bug reports. This includes using false and/or fabricated evidence/information.",
+
+         "(Rules about applications)",
+         "• Do NOT ask department recruiters, department command staff or staff members to review your application.",
+         "• Do NOT send an application to a department if you already have an application pending for that department.",
+         "• Do NOT lie or make false statements in applications, such as lying about your age or previous TXDPSRP experience/history.",
+
+            "Staff report: https://txnrp.com/r/staffReport",
+            "Player report: https://txnrp.com/r/report",
+            "Bug report: Link https://txnrp.com/r/bug",
```

Remove the comma on the last line of every list, and add missing ``]`` to close the list
```suggestion
            "Bug report: Link https://txnrp.com/r/bug"
             ]
```